src/Model/Context.php

```diff
@@ -98,11 +117,11 @@ public function has($name)
     }
 
     /**
-     * @return array
+     * @return array|\stdClass
      */
     public function jsonSerialize()
     {
-        return $this->data;
+        return $this->data ?: new \stdClass;
```

Comment:
I not a big fan of this. Why not just check if the context is not empty before send it like in my PR [here][1]? 

[1]: https://github.com/tgallice/wit-php/pull/21/files#diff-c48259b5dcb76603c5065512f13967f0R41